profile/README.md

```diff
@@ -0,0 +1,32 @@
+_API Integrations the right way_
+
+[![Discord](https://img.shields.io/discord/819563244418105354?logo=discord&logoColor=fff)](https://sfc.is/discord)
+[![NPM](https://img.shields.io/twitter/url?style=social&url=https%3A%2F%2Ftwitter.com%2Fsuperfaceai)](https://twitter.com/superfaceai)
+
+## Welcome to Superface
+
+API integrations take forever to build. They’re brittle and hard to test. APIs can change on you without notice and break your integrations.
+
+Superface is trying to change this. If your Node.js app depends on multiple API integrations, Superface is right tool for you!
+
+Our motivation behind Superface is nicely described in this [video](https://www.youtube.com/watch?v=BCvq3NXFb94) from APIdays conference.
+
+## How it works
+
+Superface is a language and protocol for abstracting integrations to application use cases. It allows use case discovery and distribution of integration code at runtime.
```

Comment:
Just little fixes (personally I find `use-case` more readable over `use case` but didn't want to break your style):

```suggestion
Superface is a language and a protocol for abstracting API integrations to application use cases. It provides use cases discovery and distribution of integration code at runtime.
```

Here's a bit different wording with less fancy terms, does it still make sense? Feel free to pick whatever you like here.

```suggestion
Superface is a language and a protocol for turning API integrations into use cases for your business needs. With Superface you can easily discover and reuse existing use cases and update your integration code at runtime.
```

Reply:
Sorry I will take only grammatical changes.